lib/tasks/package.js

```diff
@@ -25,6 +26,8 @@ class PackageTask extends Task {
     } else {
       // Assemble the project and pass to forge
       projectPath = quickTemp.makeOrRemake(this, '-tmpPath');
+      childProcess.execSync(`chmod a+x ${projectPath}`);
```

Comment:
Just use the `tmp` module -- it accepts a `mode` option as well as a `dir` option, so something like

```javascript
let projectPath = this.get('-tmpPath');
if (!projectPath) {
  projectPath = tmp.dirSync({ dir: 'tmp', mode: <whatever> }).name;
  this.set('-tmpPath', projectPath);
}
```

should work.